### PR TITLE
fixes for a few open issues

### LINKED ISSUE #32: hide ConicNonlinearBridge
If a user passes a nonlinear solver to Pajarito conic, then we should automatically wrap it with ConicNonlinearBridge.

### LINKED ISSUE #45: default solvers
Maybe use ``MathProgBase.defaultMIPsolver``, ``MathProgBase.defaultNLPsolver``, and/or ``MathProgBase.defaultConicsolver`` if user does not specify a solver?

### LINKED ISSUE #48: print nice error if conic solver is given for NLP code
Related to #32

### LINKED ISSUE #49: is_conic_solver
Should check if the solver implements ``ConicModel`` instead of hard-coding strings

### PR DESCRIPTION
**Pull request by emreyamangil**
This PR fixes a few open issues on default solvers:

Closes #49 
Closes #48 
Closes #45 
Closes #32

### CODE REVIEW

File: test/conictest.jl [hunk 1]
```diff
@@ -5,28 +5,6 @@
 
 function runconictests(algorithm, mip_solvers, conic_solvers)
 
-facts("Empty MIP solver test") do
```
Comment (mlubin) [inline test/conictest.jl]: We should add some tests which use the default solvers

Comment (emreyamangil) [inline test/conictest.jl]: Yupp! Will convert these into default solvers test asap!

**Comment by emreyamangil** [comment]
@chriscoey no need for ConicNonlinearBridge after this!

File: src/conic.jl [hunk 2]
```diff
@@ -103,15 +103,11 @@ MathProgBase.ConicModel(s::PajaritoSolver) = PajaritoConicModel(s.verbose, s.alg
...
 
-    if m.mip_solver == nothing
-        error("MIP solver is not specified.")
-    end
-
-    if m.cont_solver == nothing
-        error("Conic solver is not specified.")
-    end
+    # Check if the cont_solver is a conic solver
+    m.is_conic_solver = (applicable(MathProgBase.ConicModel, m.cont_solver) && m.cont_solver != MathProgBase.defaultNLPsolver ? true : false)
```
Comment (mlubin) [inline src/conic.jl]: The ``?`` operator is redundant, just run
```
 m.is_conic_solver = (applicable(MathProgBase.ConicModel, m.cont_solver) && m.cont_solver != MathProgBase.defaultNLPsolver)
```

Comment (emreyamangil) [inline src/conic.jl]: Coding after beer = overthinking :)

**Pull request merged**